cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -123,6 +123,10 @@
     public static final String MANAGE_COPROCESSORS = "master.manage.hbase.coprocessors";
     public static final String CLIENT_RETRIES = "hbase.client.retries.number";
     public static final String RPC_TIMEOUT = "hbase.rpc.timeout";
+    /** Whether to use latest HBase version when HBase version is unknown */
+    public static final String HBASE_VERSION_FOR_UNKNOWN_VERSION = "hbase.version";
```

Comment:
`hbase.version` seems very generic. Should we say `hbase.compat.module.version`?